docs/080-breaking-changes.rst

```diff
@@ -49,6 +49,28 @@ New Restrictions
   3. Explicit conversions between literals and enums are only allowed if the literal can
      represent a value in the enum.
 
+* There are new restrictions on explicit type conversions. The conversion is only allowed when there
+  is at most one change in sign, width or type-category (``int``, ``address``, ``bytesNN``, etc.)
+
+  Let us use the notation ``T(S)`` to denote the explicit conversion ``T(x)``, where, ``T`` and
+  ``S`` are types, and ``x`` is any arbitrary variable of type ``S``. The conversion
+  ``uint16(int8)`` is now disallowed since the conversion changes width (8 bits to 16 bits) and sign
```

Comment:
Can you make it clear that this is just an example? It could be read as the main change.

Reply:
Updated.